### DISCUSSION TITLE
Channel estimation difference between N310 and B210!

### DISCUSSION THREAD

**Discussion by aibtw** [srsran/srsRAN_Project#541] [General Help] [answered]
Hi all, 
I was interested in some parameters from channel estimation (monitoring it manually on uplink processor), and I noticed that there is a big difference in the range of values of these parameters between the USRP B210 and N310, as well as Benetel RAN550. The figures below explain what I mean:

N310, as you see, **all values** are in the negative range. The same happens with RAN550.
![image](https://github.com/srsran/srsRAN_Project/assets/60327177/06ccd8df-b716-4469-9f9c-ed716a1efc65)

But on B210, the same parameters are in the positive range, with some negative outliers:
![image](https://github.com/srsran/srsRAN_Project/assets/60327177/b5f82302-c914-409a-a6e4-d10ca2dd94ea)

It would be expected if the gain in N310/RAN550 was low when this happened, but that's not the case. The gain was high, so I am a little confused.
Can someone help me understand the reason behind the difference?

**Comment by pgawlowicz** [comment]
hi, what are the `parameters from channel estimation` and how do you extract them?

**Reply by aibtw** [reply]
sorry for the delay. they are shown on the right of the figures in the legend.
As of how we extract them, simply dumping it to a csv file directly from the cpp files. I am reading the values using the getter functions and dumping it to an output file.

**Reply by pgawlowicz** [reply, marked as answer]
Do you use the same RX gains in both cases?

**Reply by aibtw** [reply]
You are probably right, I started playing with RX gain on N310 and the values are definitely closer to the B210 now. Thanks for the help.
